tools/workspace-plugin/src/generators/prepare-initial-release/index.spec.ts

```diff
@@ -15,11 +15,11 @@ import childProcess from 'child_process';
 import generator from './index';
 import { PackageJson, TsConfig } from '../../types';
 
-const blankGraphMock = {
+const getBlankGraphMock = () => ({
```

Comment:
this was a programatical error, the const was mutated, this makes this always clean empty object per test run